src/ngx-intl-tel-input.component.ts

```diff
@@ -25,6 +28,12 @@ export class NgxIntlTelInputComponent implements OnInit {
   }
 
   ngOnInit() {
+    
```

Comment:
When user knows which countries he want to display in the dropdown, (considering the user to be a aware developer) he can easily specify the iso codes of those countries and if he doesn't know he can google them and specify those.